plugin/evm/vm.go

```diff
@@ -1762,3 +1824,7 @@ func (vm *VM) stateSyncEnabled(lastAcceptedHeight uint64) bool {
 	// enable state sync by default if the chain is empty.
 	return lastAcceptedHeight == 0
 }
+
+func (vm *VM) IssueTx(tx *Tx, b bool) error {
+	return vm.issueTx(tx, b)
+}
```

Comment:
Can we get rid of this if it's not used anywhere?

Reply:
Yeah used this for testing, will unstage.